.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+- '6'
+script:
```

Comment:
I think:
```
language: node_js
node_js:
  - "node"
```
should suffice, it tslint is essential, add it to the "test" script.
Also when using test, npm has shortcut `npm test`.
I think travis runs `npm test` by default so you don't need scripts.